composer.json

```diff
@@ -10,11 +10,11 @@
         }
     ],
     "require": {
-        "php":  ">=5.5"
+        "php":  "^7.1 || ^8.0"
     },
     "require-dev": {
         "mockery/mockery": "^1.3",
-        "phpunit/phpunit": "^4.8.35",
+        "phpunit/phpunit": "^7.5.20 || ^9.3.8",
```

Comment:
Should allow 8.x too.

Reply:
I don't see any advantage here to allow 8.x. 7.x and 9.x covers all required php versions now.